system/jhtree/jhtree.cpp

```diff
@@ -695,18 +695,18 @@ class DelayedCacheEntryReleaser : public IRemovedMappingCallback
 typedef OwningSimpleHashTableOf<CNodeMapping, CKeyIdAndPos> CNodeTable;
 class CNodeMRUSubCache final : public CMRUCacheOf<CKeyIdAndPos, CNodeCacheEntry, CNodeMapping, CNodeTable>
 {
-    std::atomic<size32_t> sizeInMem{0};
-    size32_t memLimit = 0;
+    std::atomic<size_t> sizeInMem{0};
+    size_t memLimit = 0;
 public:
     mutable CriticalSection lock;
     RelaxedAtomic<__uint64> numHits{0};
     RelaxedAtomic<__uint64> numAdds{0};
     RelaxedAtomic<__uint64> numDups{0};
     RelaxedAtomic<__uint64> numEvicts{0};
 
-    size32_t setMemLimit(size32_t _memLimit)
+    size_t setMemLimit(size_t _memLimit)
     {
-        size32_t oldMemLimit = memLimit;
+        size_t oldMemLimit = memLimit;
```

Comment:
Is there any use to check if _memLimit == memLimit and bypass this work if true?

Reply:
This is called so infrequently, that I doubt it is worth it.